app/styles/app.scss

```diff
@@ -141,6 +141,9 @@
   padding: 8px 17px;
 }
 
+// hides google optimize
```

Comment:
safe to remove?

Reply:
i think so... but i'd like kelly/raquib to take a look first before i take it out. I causes the page to go blank for a little bit if i keep it in, and seem to be no negative effects without it, but i'd like to have qa take a look before i take it out to be sure